Student/Student/Submissions/SubmissionRubric/RubricCircleView.swift

```diff
@@ -105,6 +105,7 @@ class RubricCircleView: UIView {
             }
             let button = DynamicButton(frame: CGRect(x: center.x, y: center.y, width: circleWidth, height: RubricCircleView.w))
             button.tag = i
+            button.isExclusiveTouch = true
```

Comment:
What does this do?

Reply:
Makes it so you can’t press multiple buttons at once. I think you found this one.

Reply:
Nice!